README.md

```diff
@@ -503,31 +604,31 @@ The following UCAN fragment would be valid to invoke as `did:key:zH3C2AVvLMv6gmM
 
-Except for rights amplification (below), each capability delegation MUST have equal or narrower capabilities from its witnesses. The time bounds MUST also be equal to or contained inside the time bounds of the witnesses' time bounds. This lowering of rights at each delegation is called "attenuation."
+Except for rights amplification (below), each capability delegation MUST have equal or narrower capabilities from its proofs. The time bounds MUST also be equal to or contained inside the time bounds of the proof's time bounds. This lowering of rights at each delegation is called "attenuation."
 
 ## 5.4 Rights Amplification
 
-Some capabilities are more than the sum of their parts. The canonical example is a can of soup and a can opener. You need both to access the soup inside the can, but the can opener may come from a completely separate source than the can of soup. Such semantics MAY be implemented in UCAN capabilities. This means that validating particular capabilities MAY require more than one direct witness. The relevant witnesses MAY be of a different resource and action from the amplified capability. The delegated capability MUST have this behaviour in its semantics, even if the witnesses do not.
+Some capabilities are more than the sum of their parts. The canonical example is a can of soup and a can opener. You need both to access the soup inside the can, but the can opener may come from a completely separate source than the can of soup. Such semantics MAY be implemented in UCAN capabilities. This means that validating particular capabilities MAY require more than one direct proof. The relevant proofs MAY be of a different resource and action from the amplified capability. The delegated capability MUST have this behavior in its semantics, even if the proofs do not.
 
 ## 5.6 Content Identifiers
 
-UCANs MAY be referenced by content ID (CID), per the [multiformats/cid](https://github.com/multiformats/cid) specification. The resolution of these addresses is left to the implementation and end-user, and MAY (non-exclusively) includes the following: local store, a distributed hash table (DHT), gossip network, or RESTful service.
+A UCAN token MUST be referenced as a [base32](https://github.com/multiformats/multibase/blob/master/multibase.csv#L12) [CIDv1](https://docs.ipfs.io/concepts/content-addressing/#identifier-formats), using either the [`0x55` raw data](https://github.com/multiformats/multicodec/blob/master/table.csv#L39) or [`0x0129` `dag-json`](https://github.com/multiformats/multicodec/blob/master/table.csv#L104) multicodec. Note that if using `dag-json`, the UCAN MUST be formatted as `dag-json` prior to signing.
```

Comment:
I like using CIDs for this, but is there an actual outline for how the CID would be generated over a signed `dag-json`-type UCAN? it would need to include the header and signature as fields alongside the payload, e.g.
`{
    "header": { ... },
    "payload": { ... },
    "signature": { "/": { "bytes": "..." } }
}`
but afaict this is not specified anywhere. To check my understanding, there are 2 cases:

1. Raw encoding: `sigInput = base64(JsonString(header)) + '.' + base64(JsonString(payload))`, `JWT = sigInput + '.' + base64(signature)`, and `Cid = cidv1('raw', JWT)`
2. dag-json encoding: `sigInput = base64(dagJson(header)) + '.' + base64(dagJson(payload))`, `JWT = sigInput + '.' + base64(signature)`, and `Cid = cidv1('dag-json', { header, payload, signature })` (the Cid construction is a guess)

The material difference between encodings is ordering of the fields and the encoding of Cids (e.g. `prf` elements) as strings or as objects.